src/Halogen/Svg/Attributes.purs

```diff
@@ -69,7 +69,8 @@ import Prelude
 import Data.Maybe (Maybe(..))
 import Data.Newtype (un)
 import Data.String (joinWith)
-import Halogen.HTML.Core (Prop, AttrName(AttrName), Namespace(Namespace), ClassName(..))
+import Halogen.HTML.Core (AttrName(AttrName))
```

Comment:
Since `Core` is already being introduced via qualified module, why not do the same with `AttrName` like `NameSpace`?

Reply:
The original thought was that AttrName is so heavily used that qualifying it would make the code harder to read and cause other formatting issues. But the confusion caused by some qualified and some unqualified imports from the same module is probably worse and it didn't actually create any other formatting issues (just a line that is now 82 chars, which I've left unwrapped).

Fixed via c1b34ec44a7c76d60836912e2e53af3ba6a7dd9f